### DISCUSSION TITLE
recency tilted lin_reg using kalman

### DISCUSSION THREAD

**Discussion by microprediction** [online-ml/river#1611] [Ideas]
Hi @MaxHalford been a while!  

I was thinking of banging out a recency-tilted version of linear regression (using KF formulation) but stop me if that's already in there somewhere. 

-Peter

https://github.com/online-ml/river/blob/main/river/linear_model/lin_reg.py

**Comment by MaxHalford** [comment]
Hey Peter! Yes it's been a while, I hope you're well.

We don't have that one per say, so go for it. What we do have is a way to make LR adaptive by toying with the optimizer. I would love to see what you come up with.
